UI/Services/TuitionPartnerShortlistStorage/CookieBasedTuitionPartnerShortlistStorage.cs

```diff
@@ -84,6 +84,16 @@ public void RemoveTuitionPartner(string shortlistedTuitionPartnerSeoUrl)
     public void RemoveAllTuitionPartners() =>
         _httpContextAccessor.HttpContext?.Response.Cookies.Delete(CookieName);
 
+    /// <inheritdoc/>
+    public bool IsTuitionPartnerShortlisted(string tuitionPartnerSeoUrl)
```

Comment:
Issue when testing the js disabled version...
If you go in to a TP detail, then check the checkbox (if it's not already checked) and keep pressing the "update shortlist" button over and over then it adds multiple versions of the TP to the cookie.  Meaning that the number of shortlisted TPs goes up when it shouldn't.
I think that the CookieBasedTuitionPartnerShortlistStorage.AddTuitionPartnerToCookie (and maybe AddTuitionPartnersToCookie) should only add the TP if it doesn't already exist?

Reply:
I think you are right. I was working under the impression no duplicates should ever need to be passed in.
I would change it to pick just distinct entries in `AddTuitionPartnersToCookie` 👍